packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/process/dq-response.ts

```diff
@@ -64,7 +61,6 @@ function getCreationTime(time: string | undefined): string | undefined {
   try {
     return time ? dayjs.utc(time).toISOString() : undefined;
   } catch (error) {
-    log(`Error parsing creation time: ${time}, error: ${error}`);
```

Comment:
overwhelming logs, but turns out they were just undefined values ...
mystery solved

Reply:
But if they were undefined, wouldn't the ternary work and we return `undefined`, without getting into the `catch`? Just trying to understand what the problem/solution was/is, and whether we just want to silently ignore these, with no possibility to identify opportunities of improvement.

_(I do agree that flooding logs is not a good idea, but if that's the case, we could log a percentage of the errors so we can learn)_

Reply:
So the reason was because we were converting undefined to a String, so it was String(undefined) which is "Undefined" and that was getting passed into the function. I fixed that "bug"